source/index.html.md

```diff
@@ -4429,6 +4429,187 @@ Get a single payment by its reference
 |---|---|---|---|
 |200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|OK|[GetAPaymentResponse](#schemagetapaymentresponse)|
 
+<h1 id="Zepto-API-Payouts">Payouts</h1>
```

Comment:
I noted you mix various naming conventions for html id's in this PR - 
```html
# see line 4432
<h1 id="Zepto-API-Payouts">Payouts</h1>

# vs 4446
<a id="opIdVoidAPayment"></a>

```

do we have convention for `id`'s? this is a rabbithole discussion, but some interesting points are [outlined here](https://stackoverflow.com/questions/6028211/what-is-the-standard-naming-convention-for-html-css-ids-and-classes)

is having `-` might pose a risk / ambiguity when working with `js`?
perhaps `zeptoApiPayouts` or `zepto_api_payouts` would be a better alternative? 

worth RFC @corroded, or this was agreed already?

Reply:
this is autogenerated so we don't need to worry about it